src/bin/parol/tools/templates/main.rs.tpl

```diff
@@ -1,5 +1,10 @@
 extern crate parol_runtime;
 
+// auto generation needs derive_builder
+mod derive_builder {
+    pub use parol_runtime::derive_builder::*;
+}
```

Comment:
To remove this, I've opened https://github.com/colin-kiegel/rust-derive-builder/issues/274.